### DISCUSSION TITLE
Row saved message

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6540] [Help] [answered]
how can i hide this message

![Sans titre](https://user-images.githubusercontent.com/106697208/177130302-a18d7160-f1e8-4bee-acdd-9ee6a9578690.png)

**Comment by melohagan** [comment, marked as answer]
Nope! 
Also I think it would be bad practice not to let the user know that a row had been saved.

**Reply by R0Wi** [reply]
Disabling the message is now possible https://github.com/Budibase/budibase/pull/8667

**Comment by rafikApp** [comment]
Ok. 
Can i change the message ( translate it ) ?

**Reply by melohagan** [reply]
Not at the moment, and there are no plans for this in the near future that I could see, but feel free to raise a feature request!
